yarn.lock

```diff
@@ -2753,6 +2753,16 @@ __metadata:
   languageName: node
```

Comment:
Hey)

This file contains versions, hashes and soruces for every dependency you have. It should not be changed if dependency list was not changed

(So if no package.json was changed -> this file should not be changed too)

I suggest you do this:

1. git fetch and rebase on master
2. check all submodules are inited (`git submodule update --init --recursive ) from root
3. manually delete `node_modules` folder and reinstall dependencies

Reply:
Result should be like this:

Your `yarn.lock` is 100% like `yarn.lock` on server